### ISSUE TITLE
Looking for another maintainer

### ISSUE THREAD

**Issue by jantman** [jantman/awslimitchecker#500]
I've been going through some personal difficulties lately, and it's clear from the number of open issues and PRs on this project that I don't currently have the time to dedicate to awslimitchecker that it deserves, especially given all of the recent activity and interest.

If there's anyone who's familiar with the project, has contributed before, and willing uphold the 100% code coverage requirement for unit tests, please contact me. The development documentation should be pretty complete, though the release process is pretty wonky. To begin with, I'd ask someone to stage the next release in a branch, more or less ready-to-go, and I'd kick off the release. Assuming that goes well, we'll talk about commit access.

**Comment by robpickerill** [comment]
Hi @jantman - I'm sorry to hear this. Let me take the opportunity to thank you for all your effort here.

I'm just checking in, is this still the case - your looking for support? I'd be happy to help here in any way I can. I've been looking for something to monitor AWS Limits, and this has ticked a lot of boxes for me. I've been working on a fork, but I'd be happy to help here.

**Comment by jantman** [comment]
@robpickerill Rob, I'm *definitely* still looking for help with this. Lots of things have come together in the past year that have prevented me from working more on it, which I do feel bad about. But I've been spending most of my personal time on other, non-software interests, and my current day job has made a lot of progress moving to a bunch of much smaller AWS accounts, so we have less and less use for this project.

I'd be *very* happy for any help that you could offer. For starters, if you have stuff in a fork, I'd be very interested in getting that merged and released. Beyond that, the main thing that I could really use some help with is getting some of the current PRs merged to a branch that's ready for release.

**Comment by robpickerill** [comment]
> Beyond that, the main thing that I could really use some help with is getting some of the current PRs merged to a branch that's ready for release.

Let me see if I can get that done over the week, and prepare some PR's up. Leave with me and thanks again!

**Comment by robpickerill** [comment]
Hey Jason, apologies I missed last week, got sidetracked with work and other items. Awesome, you've merged PR's - is there anything you need help with?

**Comment by jantman** [comment]
> Hey Jason, apologies I missed last week, got sidetracked with work and other items. Awesome, you've merged PR's - is there anything you need help with?

No worries! Um... there's still a handful of issues open, that I really haven't even looked at...

**Comment by oksanalipkovska** [comment]
Hi guys, hope it's the correct place to ask, with new CloudFront checks how do I override limit for already increased entries? Awslimitchecker doesn't get the info that it was increased from AWS, for example CloudFront/Cache behaviors per distribution        (limit 25) WARNING: E4J4RN2GZQXXX=22, current limit is 30, how do I override it just for this distributions and not for all of them so i can still get correct info about others?

**Comment by jantman** [comment]
@oksanalipkovska No, this is not the right place to ask that. But awslimitchecker does not currently support overriding a limit on a per-resource basis. To do that, you'd need to write a wrapper script and suppress that yourself; please see https://awslimitchecker.readthedocs.io/en/latest/python_usage.html

**Comment by oksanalipkovska** [comment]
> @oksanalipkovska No, this is not the right place to ask that. But awslimitchecker does not currently support overriding a limit on a per-resource basis. To do that, you'd need to write a wrapper script and suppress that yourself; please see https://awslimitchecker.readthedocs.io/en/latest/python_usage.html

Thank you @jantman , I'll check it out

**Comment by avazhnov-griddynamics** [comment]
As I see, @sean-zou is pretty active in https://github.com/mulesoft-labs/awslimitchecker.
But there is no "issues", only merge requests tab.

**Comment by vigneshwerv** [comment]
@jantman: Are you still looking for a maintainer? I've used this tool in the past (thank you for maintaining it thus far!) and plan to utilize it in my current job as well, although it's more oriented towards the serverless architecture.

**Comment by jantman** [comment]
@vigneshwerv Yes, I am. If you're interested, reviewing/triaging some of the current PRs and Issues would be a wonderful first step!

**Comment by Pokom** [comment]
@jantman we use this tool internally at Grafana and would like to see if we can get someone involved as a maintainer. We're not quite sure who yet, @iainlane has issued #584 that could use some attention. 

I'd be more then happy to take this conversation offline to talk through the different options and how much assistance you can provide 😄

**Comment by jantman** [comment]
@Pokom I might be able to take a peek at that issue next week.

I'm happy to take the convo offline as well, but the bottom line is that (as evidenced from the lack of activity on this project) I'd really like to find someone to completely hand over maintenance, and possibly ownership, of this project to. This was a personal-time venture for me that was previously heavily utilized at my day job. We've split most of our stuff out to many smaller accounts so service limits are much less of a concern for us, and my personal-time interest in this is essentially zero at this point.

**Comment by derrix060** [comment]
Hello, I really like the project and I'll be embarking at improving it a lot.  I have some ideas in the background where I want to take the project next (for example implemenging async in order to improve the performance, being able to query multiple regions at the same time and so on).

I was looking at a few issues and replying to some PRs, and I also noticed the Projects tab, so I just got the #384 issue to work on and I have a PR coming up soon: https://github.com/jantman/awslimitchecker/compare/master...derrix060:awslimitchecker:feature/display_percentage_on_report?expand=1 (I'll work on getting the tests passing on my end)

**Comment by Aeres-u99** [comment]
@jantman Thank you for the great tool, I will also start looking into the Open Issues, and see if I can raise few MR/PR for the same, I am looking to be the maintainer as well.
